Examples/SPM+MainBundle/Example/l10n/Unstringified.generated.swift

```diff
@@ -0,0 +1,2 @@
+// Generated by Unstringify.
```

Comment:
I guess it's normal that it's empty ?

Reply:
I added it empty because this way, if the code generation works, you can see how it is not empty anymore.
If I didn't track the file at all, the first pod install would not add the generated file to the target.